__tests__/api.test.js

```diff
@@ -57,3 +57,64 @@ describe("/api", () => {
         });
     });
 });
+
+describe("/api/articles/:article_id", () => {
+    describe("GET", () => {
+        test("200: status code of 200 is sent", () => {
+            return request(app).get("/api/articles/1").expect(200);
+        });
+        test("200: responds with an article object, which has appropriate properties", () => {
+            return request(app)
+                .get("/api/articles/1")
+                .expect(200)
+                .then(({ body: { article } }) => {
+                    expect(typeof article).toBe("object");
+                    expect(article).toEqual(
+                        expect.objectContaining({
```

Comment:
Really nice that you are checking you get back the article you are requesting rather than just expecting certain types for the values - keep up the good work!